eh_win32_msvc.cc

```diff
@@ -51,6 +56,8 @@ struct _MSVC_ThrowInfo
 	unsigned long pCatchableTypeArray;
 };
 
+extern "C" LONG WINAPI _objc_unhandled_exception_filter(struct _EXCEPTION_POINTERS* exceptionInfo);
```

Comment:
Does this actually need to be `extern`?  It shouldn't be referenced anywhere other than this compilation unit and then passed by a function pointer, so `WINAPI` ought to be sufficient.

Reply:
Good point.